/**:
  ros__parameters:
    # update frequency (Hz)
    rate: 30
    # time to wait before automatically stopping wheels after last issued command (s)
    stop_timeout: 1.0
    # radius of the robot in meters; Used to calculate wheel angles from angular velocity (m)
    robot_radius: 0.5
    # maximum wheel velocity (m/s)
    max_wheel_vel: 1.0
    # wheel acceleration (m/s^2)
    wheel_accel: 0.5
    # angular speed of the swivel when changing wheel orientation; Predicates for how long the robot will wait when adjusting wheels (rad/s)
    wheel_turn_vel: 1.57
    # minimum difference between current and target wheel swivel angle to adjust wheels in place (rad)
    max_wheel_turn_diff: 0.4
    # maximum difference between current and target wheel swivel angle to stop adjusting wheels in place (rad)
    min_wheel_turn_diff: 0.1
